roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -10,11 +10,9 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     tags = ["preflight"]
 
     def run(self, tmp, task_vars):
-        rpm_prefix = get_var(task_vars, "openshift", "common", "service_type")
-        openshift_release = get_var(task_vars, "openshift_release")
-
         args = {
-            "prefix": rpm_prefix,
-            "version": openshift_release,
+            "prefix": get_var(task_vars, "openshift", "common", "service_type"),
+            # this normalizes openshift_release (from the inventory, or defaults in openshift_facts):
```

Comment:
Actually one thing I need to pin down today is expectations about specifying the release. None are documented in the advanced install guide, and the ansible code has a mishmash of `openshift_release`, `openshift_version`, and `openshift_image_tag` as well as `deployment_type` (all undocumented).